### PR TITLE
Improve restclient recipes

### PR DESCRIPTION
**Pull request by Crandel**

### CODE REVIEW

File: recipes/know-your-http-well.rcp [hunk 1]
```diff
@@ -2,4 +2,6 @@
        :description "HTTP headers, media-types, methods, relations and status codes, all summarized and linking to their specification."
        :website "https://github.com/for-GET/know-your-http-well"
        :type github
-       :pkgname "for-GET/know-your-http-well")
+       :pkgname "for-GET/know-your-http-well"
+       :load-path ("emacs")
+       :load ("emacs/know-your-http-well.el"))
```
Comment (npostavs) [inline recipes/know-your-http-well.rcp]: I see that there are `;;;###autoload` cookies, so the explicit load shouldn't be necessary.

Comment (Crandel) [inline recipes/know-your-http-well.rcp]: There are no *.el files in root directory, so nothing load from there, except explicit load from recipe

Comment (npostavs) [inline recipes/know-your-http-well.rcp]: The `:load-path` that you added looks correct, I just don't see why you need the `:load`, everything should be autoloaded.

Comment (Crandel) [inline recipes/know-your-http-well.rcp]: Fixed

**Pull request merged**